autopush/db.py

```diff
@@ -525,11 +655,15 @@ def fetch_timestamp_messages(
         else:
             sortkey = "01;"
 
-        results = list(self.table.query_2(uaid__eq=hasher(uaid.hex),
-                                          chidmessageid__gt=sortkey,
-                                          consistent=True, limit=limit))
+        response = self.table.query(
+            KeyConditionExpression=Key('uaid').eq(hasher(uaid.hex))
+            & Key('chidmessageid').gt(sortkey),
```

Comment:
nit: maybe indent so only the keys line up.

Reply:
flake8 complains about that, but I can move the & to the previous line.

Reply:
can also wrap the whole statement in parens if that helps